code/postgres/src/main/scala/postgres/clients/PostgresClient.scala

```diff
@@ -152,6 +216,17 @@ trait UsingPostgres {
     convertJsonFieldsToPreparedStatement(insertStatement, jsonFields)
   }
 
+  private def createParentTableQueriesTSDB(jsonFields: Map[String, Any])(
```

Comment:
How about we move this function to a different file since this file is getting bloated.